rgbd/CMakeLists.txt

```diff
@@ -39,60 +38,64 @@ include_directories(
 file(GLOB_RECURSE HEADER_FILES include/*.h)
 
 add_library(rgbd    src/Server.cpp
-                    src/ServerShmOnly.cpp
+                    src/Server_RGBD.cpp
```

Comment:
This mixture of underscores and capitals looks kind of ugly. What's the convention on file names?

Reply:
We will now use the `my_class.cpp` convention.